.pre-commit-config.yaml

```diff
@@ -0,0 +1,63 @@
+---
+repos:
+  - repo: git://github.com/Lucas-C/pre-commit-hooks
+    rev: v1.1.1
+    hooks:
+      - id: remove-tabs
+
+  - repo: git://github.com/pre-commit/pre-commit-hooks
+    rev: v2.0.0
+    hooks:
+      - id: trailing-whitespace
+      - id: check-merge-conflict
+      - id: end-of-file-fixer
+      - id: name-tests-test
+      - id: check-added-large-files
+      - id: check-byte-order-marker
+      - id: check-case-conflict
+      - id: check-docstring-first
+      - id: check-json
+      - id: check-symlinks
+      - id: detect-private-key
+      - id: check-ast
+      - id: debug-statements
+
+  - repo: git://github.com/pycqa/pydocstyle.git
+    rev: 5.1.1
+    hooks:
+      - id: pydocstyle
+        exclude: '^tests*'
```

Comment:
My intention here to exclude `tests` directory from checking by `pydocstyle`, not sure whether it is the correct pattering matching?

Reply:
Looks like it could work :+1: